### ISSUE TITLE
outline format suppresses all output to stdout

### ISSUE THREAD

**Issue by danielb2** [turn-project/turn#110]
I understand this may be done on purpose to keep formatting be OK, but I had a test which hung, and I didn't realize stdout was being suppressed, so for debug printing, I was getting nowhere.

Perhaps at the start of the test, there could be a message saying stdout is being suppressed? Also, it would make sense if verbose mode or something would override that.

-d

**Comment by trans** [comment]
It gets redirected and store, but should be presented when an error occurs. But if you test just hung, then that would be a problem. Sometimes it's helpful to try alternate formats, not all of them work the same.

**Comment by danielb2** [comment]
Right. But to hide stdout from the user without letting him know is what I think should be an easy fix. If I'd been informed that this particular output format was hiding it, I would have switched right away. Instead I spent time trying to figure out what the heck was going on.

Just a simple banner saying something like this would be helpful

> NOTE: The :outline output format hides stdout. If you need stdout inline, please use a different output format.

**Comment by trans** [comment]
Okay. I'll think about adding some sort of message.
